pom.xml

```diff
@@ -178,7 +178,7 @@
         <dependency>
             <groupId>com.github.tomakehurst</groupId>
```

Comment:
https://github.com/wiremock/wiremock/releases

----
Change the Maven repository groupID to org.wiremock for all artifacts built from this repository: [wiremock](https://repo1.maven.org/maven2/org/wiremock/wiremock), [wiremock-standalone](https://repo1.maven.org/maven2/org/wiremock/wiremock-standalone), [wiremock-webhooks-extension](https://repo1.maven.org/maven2/org/wiremock/wiremock-webhooks-extension/) @tomakehurst
Change Artifact IDs of wiremock-jre8 and wiremock-jre8-standalone to wiremock and wiremock-standalone @tomakehurst
Change the standalone CLI entrypoint from com.github.tomakehurst.wiremock.standalone.WireMockServerRunner to wiremock.Run ([166c3b3](https://github.com/wiremock/wiremock/commit/166c3b3586c57aece86a1b5744a1cb2d19c6f690)) @tomakehurst